app/controllers/apo_controller.rb

```diff
@@ -313,6 +334,13 @@ def redirect
 
   # check that the user can carry out this object modification
   def forbid
+    # TODO: this could be a more granular permission request, see the
+    # Dor::Permissable#can_create_apo? and Dor::Permissable#can_manage_apo? in
+    # https://github.com/sul-dlss/dor-services/blob/permissable/lib/dor/models/permissable.rb
+    #return if current_user.can_manage?(@object, 'apo')
+    #
+    # Note that this is different from User#can_manage? because it
+    # excludes current_user.is_manager
```

Comment:
i would be a little hesitant to leave this comment for two reasons:
1) it refers to a branch in dor-services for example code, but because that branch is unmerged and in flux, it's possible that the comment will fall out of date as the referenced branch changes (and it seems unlikely that changes on an unmerged branch will trigger a developer to go look for comments or code that reference it, esp outside of the codebase in question).  not that it's never right to reference other code, just that it seems like the code being referenced should be related, committed, and relatively stable (expected to stick around for a while).
2) i think it's often a good idea to try and create an actual issue rather than leave a TODO comment, though i'm certainly not opposed to the latter in all cases.  this one would be a tough call for me, and i think i'd lean towards filing a ticket for discussion (since it appears to suggest a change in functionality, rather than just an optimization or functionally identical refactoring).

also, i'm not sure whether this is an example of how you're suggesting things could look, or what, since there's not much explanation as to what this comment line means: `#return if current_user.can_manage?(@object, 'apo')`

i also just noticed that there's at least one other similar comment in a totally different location in the code.  i think that'd push me even more towards filing an issue and getting rid of these specific TODOs.